operator/pkg/util/progress.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2019 Istio Authors
```

Comment:
nit: we should be doing 2020 now. not sure if it's legit for us to update the whole repo at once.

Reply:
I think we can remove the dates. Should probably do that to the whole repo to solve this once and for all